README.md

```diff
@@ -1,5 +1,219 @@
+1.	Ensure your API key is entered into the plugins Project Settings 
+    1. ![image](./mdcontent/projectSettingsAPIKey.png "Project Settings")
+1.	Restart Unreal (UDS will not start rendering until your API key is logged in, restarting will force that).
+1.	Add the Unlimited Detail component to either a scene actor or an existing class:
+    1. ![image](./mdcontent/addComponent.png "Add component")
+    1. ![image](./mdcontent/addUDSComponent.png "Component name")
+1.	Assign a valid UDS URL to the component:
+    1. ![image](./mdcontent/url_empty.png "Empty URL")
+    1. ![image](./mdcontent/url_valid.png "Example URL")
+1.	A test asset can be accessed with the following URL: https://azmodels.vault.euclideon.com/Japan/0_1_0.uds
+1.	Set the component scale to something large and immediately obvious:
+    1. ![image](./mdcontent/scale_large.png "Large scale")
+1.	Drag the actor into the Scene to begin rendering UDS!
+    1. ![image](./mdcontent/unrealuds.png "UDS in Unreal")
+
+# Indepth guide 
+
+## UDCloud API Key
+In order to use the Unlimited Detail Unreal plugin, you require a udCloud API key.
+
+1. Navigate to https://udcloud.com/
+1. Click the sing in/sign up button in the top right:
+    ![image](./mdcontent/udcloud_Welcome.png "UDS in Unreal")
+
+1. Clicking the Sign In / Sign up button yields the following window:
+
+    ![image](./mdcontent/udcloud_login.png "UDS in Unreal")
+
+1. Create an account with any of the listed authentication methods.
+1. After reading and accepting the EULA, Login. You will be greeted with the following window: 
+
+    ![image](./mdcontent/udcloud_loggedin.png "UDS in Unreal")
+
+1. Click your display name in the top right, to expand the following drop down:
+
+    ![image](./mdcontent/udcloud_displayNameClicked.png "UDS in Unreal")
+
+1. Click "API Keys" to load the following page: 
+
+    ![image](./mdcontent/udcloud_APIKeys.png "UDS in Unreal")
+
+1. Click on the top right button: 'Create API Key'
+
+    ![image](./mdcontent/udcloud_createAPIKey.png "UDS in Unreal")
+ 
+1. Enter a display name for the key, and (optionally) specify the number of days this key will be valid for.
+1. Click **Create an API key**. It will be listed on the page like so: 
+
+    ![image](./mdcontent/udcloud_ApiKeyList.png "UDS in Unreal")
+
+1. Under the “Options” column, click the “...” button on far right.
+    ![image](./mdcontent/udcloud_ApiKeyList.png "UDS in Unreal")
+
+ 
+
+This will allow you to either delete the key, or copy the API key to your clipboard. 
+Click Copy API Key to Clipboard
+
+Once copied, you will see the following confirmation:
+
+    ![image](./mdcontent/apikey_confirmation.png "UDS in Unreal")
+ 
+The following is the result of pasting the copied clipboard API key:
+
+`eyJhcGlrZXlpZCI6IjAxMjM0NTY3OC0xMTExLTIyMjItMzMzMy00NDQ0NTU1NTY2NjYiLCJ1c2VyaWQiOiIwMTIzNDU2NzgtMTExMS0yMjIyLTMzMzMtNDQ0NDU1NTU2NjY2IiwidmVyc2lvbiI6IjEifQ==`
```

Comment:
```suggestion
The API key will now be on your clipboard- It will have a format like: 
`eyJhcGlrZXlpZCI6IjAxMjM0NTY3OC0xMTExLTIyMjItMzMzMy00NDQ0NTU1NTY2NjYiLCJ1c2VyaWQiOiIwMTIzNDU2NzgtMTExMS0yMjIyLTMzMzMtNDQ0NDU1NTU2NjY2IiwidmVyc2lvbiI6IjEifQ==`
```